.github/workflows/ci.yml

```diff
@@ -9,34 +9,48 @@ jobs:
     container:
       image: registry.fedoraproject.org/fedora:37
     steps:
-      - run: dnf install -y git make
-      - uses: actions/checkout@v4
+      - run: dnf install -y git make rpmlint
+      - name: Lint specfiles
+        uses: actions/checkout@v4
         with:
           persist-credentials: false
-      - name: Install dependencies
-        run: |
-          make test-deps
-      - name: Run linters
-        run: |
+      - run: |
           git config --global --add safe.directory '*'
-          make lint
+          rpmlint rpm_spec/*.spec
   build-rpm:
     runs-on: ubuntu-latest
-    container:
-      image: registry.fedoraproject.org/fedora:37
+    env:
+      BUILDER_CACHE_DIR: ${{ github.workspace }}/.qubes-builderv2-cache
+      CI_SKIP_PREREQS: 1
     steps:
-      - run: dnf install -y git make
       - uses: actions/checkout@v4
         with:
-          persist-credentials: false
-      - name: Install dependencies
+          repository: QubesOS/qubes-builderv2
+          path: qubes-builderv2
+      - uses: actions/checkout@v4
+        with:
+          path: securedrop-workstation-keyring
+      - name: Install Qubes-builderv2 dependencies
         run: |
-          make build-deps
-      - name: Build RPM
+          grep -v "^docker\.io$" ${{ github.workspace }}/qubes-builderv2/dependencies-debian.txt | xargs sudo apt install -y -
```

Comment:
(the container has docker already installed, and docker.io conflicts. After trying to uninstall/reinstall, I opted to skip it and use the docker version already in ci.)